stacker_blueprints/rds/base.py

```diff
@@ -399,14 +407,36 @@ def defined_variables(self):
         variables.update(additional)
         return variables
 
+    @property
+    def master_username(self):
+        variables = self.get_variables()
+        if self.is_snapshot_restore:
```

Comment:
Hmm - you get an error when trying to set a password on a snapshot restore, right?  I think it might be preferable to raise an error early if someone tries to do that, rather than quietly ignoring their settings.  What do you think?

Reply:
You do. Added a check that user and password not be empty in this case.